add a docker file

antoine-de opened the pull request:
this way we'll be able to do

`docker run cosmogony -h`

This will make it possible to add an automated docker hub build

amatissart commented:
Hum, we need a solution so that `libpostal` submodule is accessible in the container.

amatissart commented on Dockerfile:
```diff
@@ -0,0 +1,21 @@
+
+COPY --from=builder /srv/cosmogony/target/release/cosmogony /usr/bin/cosmogony
+
+ENTRYPOINT cosmogony
```
For some reason, I think that an array `["cosmogony"]` is required in order to pass the args properly.  
Here `docker run cosmogony -h` returns an error, instead of the expected help message (easy to miss as they are similar).

antoine-de replied:
:man_facepalming:

antoine-de commented:
I've added the libpostal rules and corrected the dumb mistake of the entrypoint.

cosmogony can be run with this :+1: 
`docker run -v ~/run/osm/:/osm/ -v ~/dev/cosmogony/res:/output cosmogony -i /osm/luxembourg-latest.osm.pbf -o /output/bob.json`

(the double volumes are a bit annoying but we need to have the path to the input file and the path to the output file
